src/components/Button/src/Button.vue

```diff
@@ -194,55 +180,91 @@ export default {
 			validator: (color) => colord(color).isValid(),
 		},
 		/**
-		 * Text hover color of button (only applied on fill buttons)
+		 * How to align button's contents
 		 */
-		textColorHover: {
+		align: {
 			type: String,
 			default: undefined,
-			validator: (color) => colord(color).isValid(),
+			validator: (align) => ['center', 'stack', 'space-between'].includes(align),
 		},
 		/**
-		 * Variant
+		 * Tag of button
 		 */
-		variant: {
+		tag: {
+			type: String,
+			default: 'button',
+			validator: (tag) => ['button', 'a'].includes(tag),
+		},
+		/**
+		 * Type of the button
+		 */
+		type: {
+			type: String,
+			default: 'button',
+		},
+		/**
+		 * Toggles button disabled state
+		 */
+		disabled: {
+			type: Boolean,
+			default: false,
+		},
+		/**
+		 * Toggles button loading state
+		 */
+		loading: {
+			type: Boolean,
+			default: false,
+		},
+		/**
+		 * Advanced: MText pattern in button label
```

Comment:
Rather than prefixing the prop description with `Advanced:` can we just tag the JSDoc comment with `@advanced`? We already use tags in a few other places, notably `@public` for marking public methods or `@private` for marking private props & slots. So i think we should stick with that convention and use `@advanced` here.